wire-ios-transport/Source/Public/ZMUpdateEvent.swift

```diff
@@ -267,7 +267,9 @@ open class ZMUpdateEvent: NSObject {
     class func eventsArray(with uuid: UUID, payloadArray: [Any]?, transient: Bool, source: ZMUpdateEventSource, pushStartingAt sourceThreshold: UUID?) -> [ZMUpdateEvent] {
 
         guard let payloads = payloadArray as? [[AnyHashable: AnyHashable]] else {
-            zmLog.error("Push event payload is invalid")
+            var attributes: LogAttributes = .safePublic
+            attributes.merge([LogAttributesKey.eventId.rawValue: uuid.transportString().readableHash], uniquingKeysWith: { _, new in new })
```

Comment:
My rule of thumb would be a single argument can be one line with `[]` and multiple arguments should have breaks. I think there is even a SwiftLint rule for it, that we don't have activated yet..
```suggestion
            attributes.merge(
                [LogAttributesKey.eventId.rawValue: uuid.transportString().readableHash],
                uniquingKeysWith: { _, new in new }
            )
```